test/CadlRanchProjects/payload/multipart/src/Generated/Docs/FormData.xml

```diff
@@ -44,7 +44,7 @@ FormData client = new MultiPartClient().GetFormDataClient();
 using RequestContent content = RequestContent.Create(new
 {
     id = "<id>",
-    profileImage = File.OpenRead("<filePath>"),
+    profileImage = System.IO.File.OpenRead("<filePath>"),
```

Comment:
not sure if this is good

Reply:
this is because in the same namespace, now there is a generated model `File` and i think this is expected.